cli/muxer.c

```diff
@@ -1216,9 +1216,13 @@ static int prepare_output( muxer_t *muxer )
                         if( track_opt->ambisonic_order < 0 )
                             return ERROR_MSG( "ambisonic order must be a positive integer\n" );
                         uint32_t ambisonic_channels = (track_opt->ambisonic_order + 1) * (track_opt->ambisonic_order + 1);
+                        lsmash_boolean_t head_locked_stereo = LSMASH_BOOLEAN_FALSE;
                         /* ensure that channel count matches ambisonic order, it can also have 2 extra channels if it includes head-locked stereo */
-                        if( ambisonic_channels != summary->channels && ambisonic_channels + 2 != summary->channels )
-                            return ERROR_MSG( "ambisonic order's expected channel count does not match actual channel count\n" );
+                        if( ambisonic_channels != summary->channels )
```

Comment:
I would add braces here:

```cli/muxer.c: In function ‘prepare_output’:
cli/muxer.c:1221:27: warning: suggest explicit braces to avoid ambiguous ‘else’ [-Wdangling-else]
 1221 |                         if( ambisonic_channels != summary->channels )
      |                           ^
```

Reply:
Good point, added.